src/components/Inputs/EmailInput/EmailInput.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { EmailInput } from './index.js'
+import renderer from 'react-test-renderer'
+
+// TODO: make a helper
+// https://github.com/text-mask/text-mask/issues/427
+jest.mock('react-text-mask', () => (props) => <input type="email" {...props} />)
```

Comment:
Non-Blocking:
sorry about this :( hope it didn't waste too much time. I/we need to make a helper like my comment states :)

Reply:
I switched off `TextMaskedInput`, so I took this out